--- configure.php	2023-02-05 11:17:55.243423312 +0100
+++ configure_new.new	2023-02-05 11:16:49.313865828 +0100
@@ -821,9 +821,9 @@
           unset($_POST);
 
 	  // Over-write the config files with the clean copies
-	  exec('sudo unzip -o /usr/local/bin/config_clean.zip -d /etc/');
-	  exec('sudo rm -rf /etc/dstar-radio.*');
-	  exec('sudo rm -rf /etc/pistar-css.ini');
+	  //exec('sudo unzip -o /usr/local/bin/config_clean.zip -d /etc/');
+	  //exec('sudo rm -rf /etc/dstar-radio.*');
+	  //exec('sudo rm -rf /etc/pistar-css.ini');
 	  exec('sudo git --work-tree=/usr/local/sbin --git-dir=/usr/local/sbin/.git update-index --assume-unchanged pistar-upnp.service');
 	  exec('sudo git --work-tree=/usr/local/sbin --git-dir=/usr/local/sbin/.git reset --hard origin/master');
 	  exec('sudo git --work-tree=/usr/local/bin --git-dir=/usr/local/bin/.git reset --hard origin/master');
@@ -4886,6 +4886,7 @@
     <td align="left"><a class="tooltip2" href="#"><?php echo $lang['mmdvm_display'];?>:<span><b>Display Type</b>Choose your display type, if you have one.</span></a></td>
     <td align="left" colspan="2"><select name="mmdvmDisplayType">
 	    <option <?php if (($configmmdvm['General']['Display'] == "None") || ($configmmdvm['General']['Display'] == "") ) {echo 'selected="selected" ';}; ?>value="None">None</option>
+	    <option <?php if ($configmmdvm['General']['Display'] == "CAST") {echo 'selected="selected" ';}; ?>value="CAST">CAST</option>
 	    <option <?php if (($configmmdvm['General']['Display'] == "OLED") && ($configmmdvm['OLED']['Type'] == "3")) {echo 'selected="selected" ';}; ?>value="OLED3">OLED Type 3</option>
 	    <option <?php if (($configmmdvm['General']['Display'] == "OLED") && ($configmmdvm['OLED']['Type'] == "6")) {echo 'selected="selected" ';}; ?>value="OLED6">OLED Type 6</option>
 	    <option <?php if ($configmmdvm['General']['Display'] == "Nextion") {echo 'selected="selected" ';}; ?>value="Nextion">Nextion</option>
