--- header-menu.org	2023-02-05 10:56:25.980746922 +0100
+++ header-menu.inc	2023-02-05 11:01:08.021675373 +0100
@@ -133,6 +133,8 @@
 			<!-- Services Sub -->
 			<li class="subs has-subs"><a href="#">Pi-Star Services</a>
 			    <ul class="dropdown">
+				<li class="subs"><a href="/admin/fw/index.php">Upgrade Firmware</a></li>
+				<li class="subs"><a href="/admin/memlist/memlist.php">Memory List</a></li>
 				<li class="subs"><a href="services.php?action=stop">Stop Services</a></li>
 				<li class="subs"><a href="services.php?action=fullstop">Full Stop Services</a></li>
 				<li class="subs"><a href="services.php?action=restart">Restart Services</a></li>
